tests/test_meshtastic_utils_message_paths.py

```
@@ -245,7 +255,7 @@ def test_on_meshtastic_message_channel_fallback_numeric_portnum(
     mock_relay.assert_awaited_once()
 
 
-def test_on_meshtastic_message_unknown_portnum_logs_debug(
+def test_on_meshtastic_message_unknown_portnum_plugin_only(
     reset_meshtastic_globals,
```

Ruff (0.15.9)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)